Bergado_Shamma/style3.css

```diff
@@ -0,0 +1,28 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    background-image: url(./pictures/background.png);
+    background-attachment: fixed;
+    background-position: center;
+    background-size: cover;
+    color: #333;
+}
+
+.description, .requirement, .gallery {
+    background-color: white;
+    margin-top: 20px;
+    padding: 20px;
+}
+
+.gallery img {
+    width: 180px;
+    height: 150px;
+    margin-bottom: 10px;
+}
+
+.unordered {
+    list-style: square;
+}
```

Comment:
For the style3.css file, kindly format this with prettier. Currently it has 4 spaces before each css property. It should only have two spaces for indentation.